Dockerfile

```diff
@@ -47,20 +47,19 @@ RUN apt-get update && \
 ENV PATH="${PATH}:/opt/ruby/bin:/opt/node/bin"
 
 RUN npm install -g npm@latest && \
-	npm install -g yarn && \
 	gem install bundler && \
 	apt-get update && \
 	apt-get install -y --no-install-recommends git libicu-dev libidn11-dev \
 	libpq-dev shared-mime-info
 
-COPY Gemfile* package.json yarn.lock /opt/mastodon/
+COPY Gemfile* package.json package-lock.json /opt/mastodon/
 
 RUN cd /opt/mastodon && \
   bundle config set --local deployment 'true' && \
   bundle config set --local without 'development test' && \
   bundle config set silence_root_warning true && \
 	bundle install -j"$(nproc)" && \
-	yarn install --pure-lockfile
+	npm install
```

Comment:
`npm ci` behaves the same as `yarn install --pure-lockfile`.

https://docs.npmjs.com/cli/v8/commands/npm-ci

Reply:
```
rails assets:precompile
```
requires yarn?